web/docs/project/env-vars.md

```diff
@@ -2,61 +2,165 @@
+    <EnvVar name="WASP_SERVER_URL" type="URL" isRequired={true} note="Used as your server URL. The server needs it to properly redirect users when logging in with OAuth providers like Google or GitHub." />
+    <EnvVar name="JWT_SECRET" type="String" isRequired={true} note={
+      <span>Needed to generate secure tokens. <a href="https://jwtsecret.com/generate" target="_blank" rel="noreferrer">Generate</a> a random string at least 32 characters long.
+      </span>
+    } />
+    <EnvVar name="PORT" type="Integer" isRequired={false} defaultValue="3001" note="This is where the server listens for requests." />
+</EnvVarsTable>
+
+#### SMTP Email Sender
+
+If you are using `SMTP` as your email sender, you need to provide the following environment variables:
+
+<EnvVarsTable>
+    <EnvVar name="SMTP_HOST" type="String" isRequired={true} note="The SMTP server host." />
+    <EnvVar name="SMTP_PORT" type="Integer" isRequired={true} note="The SMTP server port." />
+    <EnvVar name="SMTP_USERNAME" type="String" isRequired={true} note="The SMTP server username." />
+    <EnvVar name="SMTP_PASSWORD" type="String" isRequired={true} note="The SMTP server password." />
+</EnvVarsTable>
+
+#### SendGrid Email Sender
+
+If you are using `SendGrid` as your email sender, you need to provide the following environment variables:
+
+
+<EnvVarsTable>
+    <EnvVar name="SENDGRID_API_KEY" type="String" isRequired={true} note="The SendGrid API key." />
+</EnvVarsTable>
+
+
+#### Mailgun Email Sender
+
+If you are using `Mailgun` as your email sender, you need to provide the following environment variables:
+
+<EnvVarsTable>
+    <EnvVar name="MAILGUN_API_KEY" type="String" isRequired={true} note="The Mailgun API key." />
+    <EnvVar name="MAILGUN_DOMAIN" type="String" isRequired={true} note="The Mailgun domain." />
+    <EnvVar name="MAILGUN_API_URL" type="URL" isRequired={false} note={
+      <span>Useful if you want to use the EU API endpoint (<code>https://api.eu.mailgun.net</code>).</span>
+    } />
+</EnvVarsTable>
+
+#### Auth
+
+You can use this env while developing to skip email verification. This is useful when you are testing the email sending functionality in development:
+
+<!-- TODO: this not really Boolean, it's really a string that accepts "true" as only true value -->
+<EnvVarsTable>
+    <EnvVar name="SKIP_EMAIL_VERIFICATION_IN_DEV" type="Boolean" isRequired={false} defaultValue="false" note="If set to true, automatically sets user emails as verified in development." />
+</EnvVarsTable>
+
+
+#### Jobs
+
+<EnvVarsTable>
+    <EnvVar name="PG_BOSS_NEW_OPTIONS" type="String" isRequired={false} note={
+        <span>It's parsed as JSON. Enables you to provide <a href="../advanced/jobs#declaring-jobs">custom config</a> for PgBoss.</span>
+    } />
+</EnvVarsTable>
+
+#### OAuth Configuration
```

Comment:
Can we put this under Auth header?

Reply:
I've moved the `Auth` stuff to `Development` and now this feels better i.e. I named it `OAuth providers`.